pkg/state/state_sriov_dp.go

```diff
@@ -81,7 +81,15 @@ func (s *stateSriovDp) Sync(customResource interface{}, infoCatalog InfoCatalog)
 	if cr.Spec.SriovDevicePlugin == nil {
 		// Either this state was not required to run or an update occurred and we need to remove
 		// the resources that where created.
-		log.V(consts.LogLevelInfo).Info("Device plugin spec in CR is nil, no action required")
+		log.V(consts.LogLevelInfo).Info("SRIOV Device plugin spec in CR is nil, deleting existing objects if needed")
```

Comment:
This whole logic repeats itself across states.

WDYT about putting it in stateSkel ?

then in logs, just use `state.name` ?

Reply:
Done